deploy/examples/operator-openshift.yaml

```diff
@@ -21,7 +21,8 @@ priority:
 allowedCapabilities: ["MKNOD"]
 allowHostIPC: true
 readOnlyRootFilesystem: false
-requiredDropCapabilities: []
+# drop all default privileges
+requiredDropCapabilities: ["All"]
```

Comment:
this will drop all the default capabilities which might be needed for the container.

https://docs.openshift.com/container-platform/3.11/admin_guide/manage_scc.html#creating-new-security-context-constraints

You can check which specific is failing and just drop that, as some capabilites would be needed or if you are dropiing all then you should add what all are needed in our code for the container.

Reply:
This will need to be well-tested on openshift. Removing privileges can be a breaking change.